npm-shrinkwrap.json

```diff
@@ -1,7395 +1,7258 @@
+      "integrity": "sha1-SlKCrBZHKek2Gbz9OtFR+BfOkfU=",
       "dev": true
     },
     "angular": {
       "version": "1.6.3",
-      "from": "angular@1.6.3",
-      "resolved": "https://registry.npmjs.org/angular/-/angular-1.6.3.tgz"
+      "resolved": "https://registry.npmjs.org/angular/-/angular-1.6.3.tgz",
+      "integrity": "sha1-XTS3mSNOj6F8ajoU4CWHM5NfQ+c="
     },
     "angular-if-state": {
       "version": "1.0.0",
-      "from": "angular-if-state@>=1.0.0 <2.0.0",
       "resolved": "https://registry.npmjs.org/angular-if-state/-/angular-if-state-1.0.0.tgz",
+      "integrity": "sha1-NqQ4N88nHs73F/bJCw3/L76WTOQ=",
       "dependencies": {
         "angular-ui-router": {
           "version": "0.3.2",
-          "from": "angular-ui-router@>=0.3.1 <0.4.0",
-          "resolved": "https://registry.npmjs.org/angular-ui-router/-/angular-ui-router-0.3.2.tgz"
+          "resolved": "https://registry.npmjs.org/angular-ui-router/-/angular-ui-router-0.3.2.tgz",
+          "integrity": "sha1-wn4EljCcmSGNVlWYWxZKCWq1IKk="
         }
       }
     },
     "angular-middle-ellipses": {
       "version": "1.0.0",
-      "from": "angular-middle-ellipses@>=1.0.0 <2.0.0",
-      "resolved": "https://registry.npmjs.org/angular-middle-ellipses/-/angular-middle-ellipses-1.0.0.tgz"
+      "resolved": "https://registry.npmjs.org/angular-middle-ellipses/-/angular-middle-ellipses-1.0.0.tgz",
+      "integrity": "sha1-zb3ECkXOMihJU1foQDV/1AyqUks="
     },
     "angular-mocks": {
       "version": "1.6.3",
-      "from": "angular-mocks@1.6.3",
       "resolved": "https://registry.npmjs.org/angular-mocks/-/angular-mocks-1.6.3.tgz",
+      "integrity": "sha1-Evr8Dx4JA/FoZABLo3W/P7kQHvE=",
       "dev": true
     },
     "angular-moment": {
       "version": "1.0.1",
-      "from": "angular-moment@1.0.1",
-      "resolved": "https://registry.npmjs.org/angular-moment/-/angular-moment-1.0.1.tgz"
+      "resolved": "https://registry.npmjs.org/angular-moment/-/angular-moment-1.0.1.tgz",
+      "integrity": "sha1-UJ0zJq6iP3giHP3bK2K/7eaEHTE="
     },
     "angular-seconds-to-date": {
       "version": "1.0.0",
-      "from": "angular-seconds-to-date@latest",
-      "resolved": "https://registry.npmjs.org/angular-seconds-to-date/-/angular-seconds-to-date-1.0.0.tgz"
+      "resolved": "https://registry.npmjs.org/angular-seconds-to-date/-/angular-seconds-to-date-1.0.0.tgz",
+      "integrity": "sha1-7eN/jrRLZXxo2XRamkBulqbefR8="
     },
     "angular-ui-bootstrap": {
       "version": "2.5.0",
-      "from": "angular-ui-bootstrap@>=2.5.0 <2.6.0",
-      "resolved": "https://registry.npmjs.org/angular-ui-bootstrap/-/angular-ui-bootstrap-2.5.0.tgz"
+      "resolved": "https://registry.npmjs.org/angular-ui-bootstrap/-/angular-ui-bootstrap-2.5.0.tgz",
+      "integrity": "sha1-L6zvuRU4ZlXcX0QVAlgDJRcjbQE="
     },
     "angular-ui-router": {
       "version": "0.4.2",
-      "from": "angular-ui-router@>=0.4.2 <0.5.0",
```

Comment:
This is what caused the earlier confusion with our ensure-staged-shrinkwrap script telling us that the shrinkwrap file had changed, even though we had the identical module versions installed.